cmd/DownloadGrantsGovDB/handler.go

```diff
@@ -23,7 +23,7 @@ type ScheduledEvent struct {
 // grantsURL returns the download URL for the Grants.gov database export.
 func (e *ScheduledEvent) grantsURL() string {
 	return fmt.Sprintf("%sGrantsDBExtract%sv2.zip",
-		env.GrantsGovBaseURL + env.GrantsGovPathURL,
+		env.GrantsGovBaseURL+env.GrantsGovPathURL,
```

Comment:
This change is nonfunctional; was applied automatically when I ran `task check` locally (which runs `go fmt`) before pushing.

Reply:
Missed that part my last PR; thanks for pointing it out!